doc/modules/cross_validation.rst

```diff
@@ -235,6 +235,20 @@ Here is an example of ``cross_validate`` using a single metric::
     >>> sorted(scores.keys())
     ['estimator', 'fit_time', 'score_time', 'test_score', 'train_score']
 
+Cross-validation significance evaluation
+----------------------------------------
```

Comment:
I would add this section after "Obtaining predictions by cross-validation"

Reply:
Done in PR #14757.